src/main/java/com/ss/ita/rozetka/pageobject/modals/BasketModal.java

```diff
@@ -93,6 +93,14 @@ public T close() {
         return pageObject;
     }
 
+    @Step("BasketModal: Add additional services")
+    public BasketModal<T> addAdditionalServices() {
+        int totalPrice = getProductsTotalPrice();
```

Comment:
add a comment why is this needed, it's not clear why you get the product price in a method that adds additional services